data/2019.ts

```diff
@@ -13,6 +19,11 @@ export default {
             link: 'https://github.com/vercel/next.js/releases/tag/v9.0.0',
             icon: 'next'
         },
+        {
+            date: '2019-06-04',
+            title: 'NPM passes 1 million packages',
+            link: 'https://snyk.io/blog/npm-passes-the-1-millionth-package-milestone-what-can-we-learn/',
+            icon: 'npm'
```

Comment:
Missing '},'

Reply:
Oops, sorry! I don't know how I missed that. Fixed now :)